lib/commands/serve.js

```diff
@@ -38,10 +39,12 @@ module.exports = Command.extend({
   ],
 
   run(commandOptions) {
+    if (commandOptions.liveReloadPort) {
```

Comment:
q: should we deprecate? Or should we merely default to the same port, but allow users to specify a custom port if they need. For example their are some users which serve the HTTP index via other systems (such as rails) and they rely on a different port.

Reply:
We can warn and continue to use the custom port. Probably it will be a good solution. Current code completely ignores the custom host as well.

Reply:
Should we warn at all?

Reply:
Nope. We should add it in the help document that it defaults to same port ember app.
Working on it. I will get that change soon.